components/suggestion/modal_suggestion_list.jsx

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SuggestionList from 'components/suggestion/suggestion_list.jsx';
+import {getClosestParent} from 'utils/utils.jsx';
+
+export default class ModalSuggestionList extends React.PureComponent {
+    static propTypes = {
+        location: PropTypes.string.isRequired,
+        open: PropTypes.bool.isRequired,
+        cleared: PropTypes.bool.isRequired,
+        calculateInputRect: PropTypes.func.isRequired,
+        onLoseVisibility: PropTypes.func.isRequired,
```

Comment:
(0/5) onHide seems like an active action. I prefer onLoseVisibility since it is more passive. The user does not actively hide the input, the input lose visibility when the user scrolls for whichever reason.